pallets/aleph/src/impls.rs

```diff
@@ -50,3 +52,9 @@ where
         }
     }
 }
+
+impl<T: Config> FinalityCommitteeManager<T::AccountId> for Pallet<T> {
+    fn next_session_finality_committee(committee: Vec<T::AccountId>) {
```

Comment:
nit: how about preppending the name with `set_*` to make it more clear we're mutating the storage? This method name could well be returning next session committee.

Reply:
But this is just a particular implementation of this trait.  The purpose of the trait is just to inform others about the new committee. The name though, hmm, maybe `on_next_session_finality_committee` would be better?

Reply:
Yeah, I know I commented on an impl (and not the trait) but it was first when I was reading the code :)

Reply:
Oh, and I like your suggestion for the name as well. Sounds a bit call-backy but that's fine.

Reply:
ok, done